mne/time_frequency/spectrum.py

```diff
@@ -369,13 +370,19 @@ def _check_values(self):
             warn(f'Zero value in spectrum for channel{s} {", ".join(chs)}',
                  UserWarning)
 
+    def _complex_tapers(self, **method_kw):
```

Comment:
Seems potentially useful more broadly. At this point my only comment is please change this to "_has_complex_tapers" or "_check_for_complex_tapers" or similar.